diagnostic.md

```diff
@@ -23,7 +37,7 @@ Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
-
+$(li).up('form');
```

Comment:
`li` needs to be declared as a selector string or an object in order for this to work.  Even then, this will still cause an error since `.up()` is not a method on jQuery objects.  A few possible ways are `$('li').parent().parent().find('form')`, `$('li').parent().parent().children('form')`, and `$('.listerItem').parent().parent().$('form')`.